vertexai/genai/clientconfig.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package genai
+
+// ClientConfig has configurations for the client.
+type ClientConfig struct {
+	// RESTTransport asks the client to use REST as its underlying transport
+	// instead of gRPC.
+	RESTTransport bool
```

Comment:
The doubled `T` is confusing. Maybe `UseREST` or just `REST`?
If you like it the way it is, that's fine too.